lib/services/project-service.ts

```diff
@@ -50,10 +51,12 @@ export class ProjectService implements IProjectService {
 
 			await this.ensureAppResourcesExist(projectDir);
 
-			let packageName = constants.TNS_CORE_MODULES_NAME;
-			await this.$npm.install(packageName, projectDir, { save: true, "save-exact": true });
-
 			let templatePackageJsonData = this.getDataFromJson(templatePath);
+
+			if (!(templatePackageJsonData && templatePackageJsonData.dependencies && templatePackageJsonData.dependencies[constants.TNS_CORE_MODULES_NAME])) {
```

Comment:
@yyosifov the template versioning may differ and we also do not have control over the templates - basically everything is template. 
Also the current code will install `tns-core-modules` only in case they are not defined in the template itself.